test/OrchardCore.Tests/Data/JsonDynamicTests.cs

```diff
@@ -793,26 +795,61 @@ public void JsonDynamicValueIsComparableToString()
     [Fact]
     public void SerializingJsonDynamicValueMustWriteValueOnly()
     {
-       // Arrange
-        var contentItem = new ContentItem();
-        contentItem.Alter<TestPart>(part =>
-        {
-            part.TextFieldProp = new TextField { Text = "test" };
-            part.NumericFieldProp = new NumericField { Value = 123 };
-            part.BooleanFieldProp = new BooleanField { Value = true };
-        });
+        // Arrange
+        var contentItem = GetContentTestData();
+
+        dynamic contentExpando = new ExpandoObject();
+        contentExpando.content = contentItem.Content;
+        var contentStr = JConvert.SerializeObject((ExpandoObject)contentExpando);
 
         // Act
+        Assert.Equal("{\"content\":{\"TestPart\":{\"TextFieldProp\":{\"Text\":\"test\"},\"NumericFieldProp\":{\"Value\":123},\"BooleanFieldProp\":{\"Value\":true}}}}", contentStr);
```

Comment:
I don't think this extra test is necessary, it does basically the same.

Reply:
I agree. It looks like it now.